lib/itax_code/omocode.rb

```diff
@@ -8,7 +8,7 @@ class Omocode
     #
     # @param [String] tax_code
     # @param [Utils]  utils
-    def initialize(tax_code, utils = Utils.new)
+    def initialize(tax_code, utils = ItaxCode::Utils)
```

Comment:
The prefix is unnecessary here, as you're already inside the ItaxCode module.

```suggestion
    def initialize(tax_code, utils = Utils)
```